integration/terraform/ec2/linux/iam.tf

```diff
@@ -44,7 +44,8 @@ data "aws_iam_policy_document" "user-managed-policy-document" {
       "logs:PutLogEvents",
       "s3:GetObjectAcl",
       "s3:GetObject",
-      "s3:ListBucket"
+      "s3:ListBucket",
+      "dynamodb:*"
```

Comment:
I would rather figure out which DDB actions we need than put * for the terraform creation